dotnet-desktop-guide/net/winforms/printing/overview.md

```diff
@@ -0,0 +1,109 @@
+
+    > [!NOTE]
+    > You can add a <xref:System.Windows.Forms.PrintDialog> component to your form to provide a clean and efficient user interface (UI) to your users. Setting the <xref:System.Windows.Forms.PrintDialog.Document%2A> property of the <xref:System.Windows.Forms.PrintDialog> component enables you to set properties related to the print document you're working with on your form. For more information about the <xref:System.Windows.Forms.PrintDialog> component, see [PrintDialog Component](/dotnet/desktop/winforms/controls/printdialog-component-windows-forms?view=netframeworkdesktop-4.8&preserve-view=true).
+
+     For more information about the specifics of Windows Forms print jobs, including how to create a print job programmatically, see <xref:System.Drawing.Printing.PrintPageEventArgs>.
+
+## How to complete print jobs
+
+Frequently, word processors and other applications that involve printing will provide the option to display a message to users that a print job is complete. You can provide this functionality in your Windows Forms by handling the <xref:System.Drawing.Printing.PrintDocument.EndPrint> event of the <xref:System.Drawing.Printing.PrintDocument> component.
+
+ The following procedure requires that you've created a Windows-based application with a <xref:System.Drawing.Printing.PrintDocument> component on it. The procedure given below is the standard way of enabling printing from a Windows-based application. For more information about printing from Windows Forms using the <xref:System.Drawing.Printing.PrintDocument> component, see [How to: Create Standard Windows Forms Print Jobs](/dotnet/desktop/winforms/advanced/how-to-create-standard-windows-forms-print-jobs?view=netframeworkdesktop-4.8&preserve-view=true).
+
+01. Set the <xref:System.Drawing.Printing.PrintDocument.DocumentName%2A> property of the <xref:System.Drawing.Printing.PrintDocument> component.
+
+    ```vb
+    PrintDocument1.DocumentName = "SamplePrintApp"
+    ```
+
+    ```csharp
+    printDocument1.DocumentName = "SamplePrintApp";
+    ```
+
+01. Write code to handle the <xref:System.Drawing.Printing.PrintDocument.EndPrint> event.
+
+     In the following code example, a message box is displayed, indicating that the document has finished printing.
+
+    :::code language="vb" source="snippets/overview/vb/Form1.vb" id="message-box-indicating-document-has-finished-printing":::
+    :::code language="csharp" source="snippets/overview/csharp/Form1.cs" id="message-box-indicating-document-has-finished-printing":::
+
+    (Visual C#) Place the following code in the form's constructor to register the event handler.
+
+    ```csharp
+    this.printDocument1.EndPrint += new System.Drawing.Printing.PrintEventHandler (this.printDocument1_EndPrint);
+    ```
```

Comment:
Don't forget to remove this
```suggestion
```

Reply:
When converted project to .NET 6.0, these lines are needed in the code. Adding this as inline code as it is just a single line.